<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>653</width>
    <height>323</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
background-color:white;
border-radius: 20px
}</string>
  </property>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>200</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QSpinBox {
    	font: 300 20pt&quot;Epilogue&quot;;  
    	background-color: white;
	border-radius:10px;
    	color: #1d1d1d;
	border: none;
}
QSpinBox QLineEdit {
    background-color: rgb(255,255,255);
    border: none;
    padding: 4px 6px;
}
/*QSpinBox::up-button, QSpinBox::down-button {
    background-color: rgb(255,255,255);
    border: 1px solid rgb(200,200,200);
    border-radius: 3px;
    min-width: 16px;
    min-height: 14px;
}
QSpinBox::up-button:hover, QSpinBox::down-button:hover {
    background-color: pink;
}*/</string>
   </property>
   <property name="wrapping">
    <bool>false</bool>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::ButtonSymbols::UpDownArrows</enum>
   </property>
   <property name="suffix">
    <string extracomment="minutes"/>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdSeconds">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>40</y>
     <width>231</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLCDNumber{
background-color:white;
color:#1d1d1d;
qproperty-segmentStyle: Filled;
border-radius:10px;
border:none;
padding: 20px;

}</string>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="digitCount">
    <number>2</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>120</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border-radius: 25px;
border: 1px solid #C0C0C0;
font: 300 10px &quot;Epilogue&quot;;
background-color: white;
color:#1d1d1d
}
QPushButton:hover{
background-color: #C0C0C0
}</string>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdMinutes">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLCDNumber{
background-color:white;
color:#1d1d1d;
qproperty-segmentStyle: Filled;
border-radius:10px;
border:none;
padding: 20px;

}</string>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="digitCount">
    <number>2</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>240</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font/>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
background-color:white;
color:#1d1d1d;
border-radius:10px;
border:none;
padding-left: 20px;

}</string>
   </property>
   <property name="text">
    <string>Seconds:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>120</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border-radius: 25px;
border: 1px solid #C0C0C0;
font: 300 10px &quot;Epilogue&quot;;
background-color: white;
color:#1d1d1d
}
QPushButton:hover{
background-color: #C0C0C0
}</string>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border-radius: 25px;
border: 1px solid #C0C0C0;
font: 300 10px &quot;Epilogue&quot;;
background-color: white;
color:#1d1d1d
}
QPushButton:hover{
background-color: #C0C0C0
}</string>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdCS">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>40</y>
     <width>211</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLCDNumber{
background-color:white;
color:#1d1d1d;
qproperty-segmentStyle: Filled;
border-radius:10px;
border:none;
padding: 20px;

}</string>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="digitCount">
    <number>2</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>240</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font/>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
background-color:white;
color:#1d1d1d;
border-radius:10px;
border:none;
padding-left: 20px;

}</string>
   </property>
   <property name="text">
    <string>Minutes: </string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>200</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QSpinBox {
    	font: 300 20pt&quot;Epilogue&quot;;  
    	background-color: white;
	border-radius:10px;
    	color: #1d1d1d;
	border: none;
}
QSpinBox QLineEdit {
    background-color: rgb(255,255,255);
    border: none;
    padding: 4px 6px;
}
/*QSpinBox::up-button, QSpinBox::down-button {
    background-color: rgb(255,255,255);
    border: 1px solid rgb(200,200,200);
    border-radius: 3px;
    min-width: 16px;
    min-height: 14px;
}
QSpinBox::up-button:hover, QSpinBox::down-button:hover {
    background-color: pink;
}*/</string>
   </property>
   <property name="wrapping">
    <bool>false</bool>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::ButtonSymbols::UpDownArrows</enum>
   </property>
   <property name="suffix">
    <string extracomment="minutes"/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
